pwndbg/ui.py

```diff
@@ -22,13 +23,17 @@
 
 def banner(title):
     title = title.upper()
-    try:
-        _height, width = struct.unpack('hh', fcntl.ioctl(sys.stdin.fileno(), termios.TIOCGWINSZ, '1234'))
-    except:
-        width = 80
+    _height, width = get_window_size()
     width -= 2
     return C.banner(("[{:%s^%ss}]" % (config.banner_separator, width)).format(title))
 
 def addrsz(address):
     address = int(address) & pwndbg.arch.ptrmask
     return "%{}x".format(2*pwndbg.arch.ptrsize) % address
+
+def get_window_size():
+    try:
+        rows, cols = struct.unpack('hh', fcntl.ioctl(sys.stdin.fileno(), termios.TIOCGWINSZ, '1234'))
```

Comment:
Why `'1234'`? If we want to pass any string, I believe we can omit the argument so the behavior will be as if a string had been passed. From `ioctl(fd, op[, arg[, mutate_flag]])` docstring:
```
(...)
If the argument is a mutable buffer and the mutable_flag argument is not
passed or is false, the behavior is as if a string had been passed.  This
behavior will change in future releases of Python.

If the argument is an immutable buffer (most likely a string) then a copy
of the buffer is passed to the operating system and the return value is a
string of the same length containing whatever the operating system put in
the buffer.  The length of the arg buffer in this case is not allowed to
exceed 1024 bytes.
```